sql/table.go

```diff
@@ -87,13 +91,21 @@ func makeTableDesc(p *parser.CreateTable) (structured.TableDescriptor, error) {
 				ColumnNames: d.Columns,
 			}
 			if d.PrimaryKey {
-				index.Name = structured.PrimaryKeyIndexName
+				if index.Name == "" {
```

Comment:
still not sure why this is here - if a user-supplied name made it into the primary key, something is wrong. I think either remove the check or panic